ps3.py

```diff
@@ -322,27 +322,68 @@ def play_game(word_list):
     word_list: list of lowercase strings
     """
     hands_number = int(input("Enter total number of hands: "))
-    hands_number_copy_for_cycle = hands_number
     score_for_the_game = 0
-    hand_match = deal_hand(HAND_SIZE)
-    replay_the_hand = 'yes'
+    replay_the_hand_switch = True
+    substitute_letter_switch = True
+    hand_without_substitute = 0
+    saved_old_score = 'no_old_score'
```

Comment:
Please - do not mix types for variable. Also - there's no need to store 'old_score', as it doesn't matter

Reply:
done